diff --git a/src/NotificationItem.php b/src/NotificationItem.php
index 4c28fde..4d52692 100644
--- a/src/NotificationItem.php
+++ b/src/NotificationItem.php
@@ -39,6 +39,12 @@ class NotificationItem {
    */
   public $icon = '';
 
+  /**
+   * @var string
+   *  The notification image.
+   */
+  public $image = '';
+
   /**
    * @var string
    *   The bundle name. What bundle is used to create this item.
@@ -71,6 +77,7 @@ class NotificationItem {
       'body' => $this->body,
       'url' => $this->url,
       'icon' => $this->icon,
+      'image' => $this->image,
     ]);
   }
 }
